samples/contrib/aws-samples/mnist-kmeans-sagemaker/mnist-classification-pipeline.py

```diff
@@ -78,187 +89,104 @@ def training_input(input_name, s3_uri):
         instance_type=instance_type,
-        instance_count=instance_count,
-        volume_size=volume_size,
-        max_num_jobs=hpo_max_num_jobs,
-        max_parallel_jobs=hpo_max_parallel_jobs,
-        max_run_time=max_run_time,
-        network_isolation=network_isolation,
-        traffic_encryption=traffic_encryption,
-        spot_instance=hpo_spot_instance,
-        max_wait_time=hpo_max_wait_time,
-        checkpoint_config=hpo_checkpoint_config,
+        max_num_jobs=3,
+        max_parallel_jobs=2,
         role=role_arn,
     ).after(process)
 
     training = sagemaker_train_op(
         region=region,
-        endpoint_url=endpoint_url,
         image=train_image,
         training_input_mode=train_input_mode,
         hyperparameters=hpo.outputs["best_hyperparameters"],
-        channels=train_channels,
+        channels=trainChannels,
         instance_type=instance_type,
-        instance_count=instance_count,
-        volume_size=volume_size,
-        max_run_time=max_run_time,
         model_artifact_path=train_output_location,
-        output_encryption_key=output_encryption_key,
-        network_isolation=network_isolation,
-        traffic_encryption=traffic_encryption,
-        spot_instance=train_spot_instance,
-        max_wait_time=train_max_wait_time,
-        checkpoint_config=train_checkpoint_config,
         role=role_arn,
     )
 
     create_model = sagemaker_model_op(
         region=region,
-        endpoint_url=endpoint_url,
         model_name=training.outputs["job_name"],
         image=training.outputs["training_image"],
         model_artifact_url=training.outputs["model_artifact_url"],
-        network_isolation=network_isolation,
         role=role_arn,
     )
 
-    prediction = sagemaker_deploy_op(
-        region=region, endpoint_url=endpoint_url, model_name_1=create_model.output,
+    sagemaker_deploy_op(
+        region=region, model_name_1=create_model.output,
     )
 
-    batch_transform = sagemaker_batch_transform_op(
+    sagemaker_batch_transform_op(
         region=region,
-        endpoint_url=endpoint_url,
         model_name=create_model.output,
-        instance_type=batch_transform_instance_type,
-        instance_count=instance_count,
-        max_concurrent=batch_transform_max_concurrent,
-        max_payload=batch_transform_max_payload,
-        batch_strategy=batch_strategy,
-        input_location=batch_transform_input,
-        data_type=batch_transform_data_type,
-        content_type=batch_transform_content_type,
-        split_type=batch_transform_split_type,
-        compression_type=batch_transform_compression_type,
-        output_location=batch_transform_ouput,
+        instance_type=instance_type,
+        batch_strategy="MultiRecord",
+        input_location=f"s3://{my_bucket_name}/mnist_kmeans_example/input",
+        content_type="text/csv",
+        split_type="Line",
+        output_location=f"s3://{my_bucket_name}/mnist_kmeans_example/output",
     )
 
 
```

Comment:
Can we target doing 150 lines and trim the excess whitespace, and review the fields again to see if there are shorter options

Reply:
I am actually fixing linting with this PR using flake8, will see if it can be done